src/core/config/config.rs

```diff
@@ -694,19 +828,19 @@ impl Config {
         Ok(serde_yaml::from_str(yaml)?)
     }
 
-    pub fn from_sdl(sdl: &str) -> Valid<Self, String> {
+    pub fn from_sdl(input_path: &str, sdl: &str) -> Valid<Self, String> {
         let doc = async_graphql::parser::parse_schema(sdl);
         match doc {
-            Ok(doc) => from_document(doc),
+            Ok(doc) => from_document(input_path, doc),
             Err(e) => Valid::fail(e.to_string()),
         }
     }
 
     pub fn from_source(source: Source, schema: &str) -> Result<Self> {
-        match source {
-            Source::GraphQL => Ok(Config::from_sdl(schema).to_result()?),
-            Source::Json => Ok(Config::from_json(schema)?),
-            Source::Yml => Ok(Config::from_yaml(schema)?),
+        match source.input_type {
+            SourceType::GraphQL => Ok(Config::from_sdl(&source.input_path, schema).to_result()?),
+            SourceType::Json => Ok(Config::from_json(schema)?),
+            SourceType::Yml => Ok(Config::from_yaml(schema)?),
```

Comment:
The position information should be available in `Json` and `YML` format also. This is what is happenening right now

```bash
cargo run -- start ./examples/jsonplaceholder.json
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.73s
     Running `target/debug/tailcall start ./examples/jsonplaceholder.json`
 INFO File read: ./examples/jsonplaceholder.json ... ok
 INFO N + 1 detected: 1
ERROR Invalid Configuration
Caused by:
  • No base URL defined [at  0#0]
  • No base URL defined [at  0#0]
  • No base URL defined [at  0#0]
  ```

Reply:
Yeah that's bad, I guess I have to find a way to get the positional details for when serde deserialize the data from json and yaml